source/hardware/controllers/reloop_mixage.rst

```diff
@@ -0,0 +1,115 @@
+Reloop Mixage
+=============
+
+The Reloop Mixage line of controllers contains 2-channel DJ controller. It offers two jog wheels, line faders, a cross-fader, controls for looping, effects, equalizers and gain controls. The Interface Edition version comes with an integrated soundcard, headphone connection and microphone input and exists in version MK1 (unmarked) and MK2 (marked on back of device)
+
+.. figure:: ../../_static/controllers/reloop_mixage_top.jpg
+   :align: center
+   :figwidth: 100%
+   :alt: Reloop Mixage IE (top view)
+   :figclass: pretty-figures
+
+   Reloop Mixage top view (© Reloop, used with permission)
+
+Audio
+-----
+
+This controller is a MIDI and audio class compliant device so it is compatible as-is with Linux, MacOS and Windows. On Windows, the manufacturer ships an :term:`ASIO` low-latency driver that can be found on the `manufacturer’s product
+page <productpage_url_>`_.
+
+The microphone :hwlabel:`🎤` input can be set to different modes. To make it usable in Mixxx, set it to **SW**. The :hwlabel:`MASTER` and :hwlabel:`PHONES` level controls work directly on the audio hardware and can't be mapped in Mixxx.
+
+Control overview
+----------------
+
+.. figure:: ../../_static/controllers/reloop_mixage_overview.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Reloop Mixage button overview
+   :figclass: pretty-figures
+
+   Reloop Mixage button overview
+
+All mapped controls use MIDI channel #1 (can be changed on the back of the device). Most functions on this controller are quite self-explanatory and mapped in that way (you might want to check the official manual too)
+
+======  =========================================  ==========================================================================================================================================================================  ==============================================================================================================================================================================
+Button  Label                                      Function                                                                                                                                                                    SHIFT function
```

Comment:
```suggestion
#  Label                                      Function                                                                                                                                                                    SHIFT function
```
This allows to shrink the column to a minimum. (and not all controls are buttons ;)